deny.toml

```diff
@@ -0,0 +1,180 @@
+# This template contains all of the possible sections and their default values
+# Note that all fields that take a lint level have these possible values:
+# * deny - An error will be produced and the check will fail
+# * warn - A warning will be produced, but the check will not fail
+# * allow - No warning or error will be produced, though in some cases a note
+# will be
+# The values provided in this template are the default values that will be used
+# when any section or field is not specified in your own configuration
+# If 1 or more target triples (and optionally, target_features) are specified,
+# only the specified targets will be checked when running `cargo deny check`.
+# This means, if a particular package is only ever used as a target specific
+# dependency, such as, for example, the `nix` crate only being used via the
+# `target_family = "unix"` configuration, that only having windows targets in
+# this list would mean the nix crate, as well as any of its exclusive
+# dependencies not shared by any other crates, would be ignored, as the target
+# list here is effectively saying which targets you are building for.
+# The triple can be any string, but only the target triples built in to
+# rustc (as of 1.40) can be checked against actual config expressions
+# { triple = "x86_64-unknown-linux-musl" },
+# You can also specify which target_features you promise are enabled for a
+# particular target. target_features are currently not validated against
+# the actual valid features supported by the target architecture.
+# { triple = "wasm32-unknown-unknown", features = ["atomics"] },
+targets = []
+
+# This section is considered when running `cargo deny check advisories`
+# More documentation for the advisories section can be found here:
+# https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html
+[advisories]
+# The path where the advisory database is cloned/fetched into
+db-path = "~/.cargo/advisory-db"
```

Comment:
Because this is a template with all the defaults set, it is quite hard to figure out what exactly was changed and what's stock. I suggest just keep the values that were actually modified from the defaults throughout.

Reply:
Done.